components/form_inputs/forms/DisabledForm.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <v-container>
+    <v-row
+
+      justify="space-between"
+    >
+      <v-col
+        cols="12"
+        md="4"
+      >
+        <v-form ref="form"
+          disabled
+        >
+          <v-text-field
+            v-model="model"
+            label="First name"
+          ></v-text-field>
+        </v-form>
+      </v-col>
+
+      <v-col
+        cols="12"
+        md="6"
+      >
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
+<script>
+export default {
+  data: () => ({
+    allowSpaces: false,
+    match: 'Foobar',
+    max: 0,
+    model: 'Foobar',
+  }),
+  computed: {
```

Comment:
Do we really need this for this form?

Reply:
Why not? To check disabled property

Reply:
I am talking about script section, it's about rules to check input, but we have disabled or readonly form so we can't type something

Reply:
Fixed